File: RZHD/ClientApp/src/app/api/models/order-view.ts
/* tslint:disable */
import { ProductView } from './product-view';
import { RestaurantView } from './restaurant-view';
import { StationView } from './station-view';
import { Status } from './status';
export interface OrderView  {
  id?: number;
  products?: null | Array<ProductView>;
  restaurants?: null | Array<RestaurantView>;
  stations?: null | Array<StationView>;
  status?: Status;
  totalPrice?: number;
}

File: RZHD/Models/Responses/Restaurants/OrderView.cs
using RZHD.Models.Responses.Stations;
using System.Collections.Generic;

namespace RZHD.Models.Responses.Restaurants
{
    public class OrderView
    {
        public int Id { get; set; }
        public Status Status { get; set; }
        public int TotalPrice { get; set; }

        public List<ProductView> Products { get; set; }
        public List<StationView> Stations { get; set; }
        public List<RestaurantView> Restaurants { get; set; }
    }
}
